### PR TITLE
feature/OMEGA-99 add emptyInvoiceBox component

### PR DESCRIPTION
**Pull request by marasva**
[OMEGA-99](https://elvia.atlassian.net/browse/OMEGA-99?atlOrigin=eyJpIjoiMTJmMTgwYWE4ODc0NDMxYzkwNTUwNWY2ZjVkOWJjM2QiLCJwIjoiaiJ9)

## What is the reason for this change?

Customers that has not yet received an invoice is shown an empty table. 
image

### What is the expected result?
A new view with text and icons explaining that the customer have not yet received an invoice.

image

## How is it solved?

Add new Begrep in contentful and add a new component with icons from Figma.

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |        |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?

- [ ] Use the test user Per Hansen and go into his meterpoint view for Ole Nilsens gate 21 (222222222)
- [ ] A new view will show with the same design as above
- [ ] Looks as usual (with a table view of invoices) if you try some of the other meterpoints.

### CODE REVIEW

**Comment by marasva** [comment]
I changed the @ media to 1120px because I could not find a way to keep the design without breaking it at the breakpoint between 1024px - 1060px through using the regular max-width for MobileAndTablet.

File: packages/hdd-react/src/components/EmptyInvoiceBox/EmptyInvoiceBox.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  style,
+}) => {
+  useStyles(css)
+  const classes = classnames(css.emptyInvoice, className)
+
+  return (
+    <>
+      <div className={classes} style={style}>
+        <div className={css.leftIconsEmptyInvoice}>
+          <div>
```
Comment (eTallang) [inline packages/hdd-react/src/components/EmptyInvoiceBox/EmptyInvoiceBox.tsx:33]: Are these `<div>` elements that wrap the icons necessary? :slightly_smiling_face: Maybe we could do something like:
```
<HddSendEmailLargeColor />
<HddSendEmailSmallColor className={css.sendEmailLeft2} />
<HddPaperPlaneOrangeSmallColor className={css.paperPlaneLeft} />
```

Comment (marasva) [reply packages/hdd-react/src/components/EmptyInvoiceBox/EmptyInvoiceBox.tsx:33]: I cannot manage to make it work without them, but I agree that it looks unnecessary

File: packages/ui/src/components/EmptyInvoiceBox/EmptyInvoiceBox.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+export const EmptyInvoiceBox: React.FC<EmptyInvoiceBoxProps> = ({
+  className,
+  style,
+  infoText,
+}) => {
+  const classes = classnames(css.emptyInvoice, className)
+
+  return (
+    <>
+      <div className={classes} style={style}>
```
Comment (JonatanPaalsson) [inline packages/ui/src/components/EmptyInvoiceBox/EmptyInvoiceBox.tsx:30]: Is it possible to remove some of these divs? Some icons are just wrapped in a empty div and maybe the classnames can go directly on the icons?

**Review by JonatanPaalsson — CHANGES_REQUESTED** [review]

**Comment by lobunto** [comment]
@marasva what is the status for this PR?

**Comment by marasva** [comment]
Well, I do not know of any "smoother" way to implement the svgs without so many divs, so for me it is finished. Do not think @JonatanPaalsson have looked further on it either?

**Comment by t0ff3r** [comment]
Seems overly complex to import and place every icon freely one by one. I am no expert on svgs but could we solve this by making a nested svg instead of handling each of them separately? 

This article seems interesting: https://www.sarasoueidan.com/blog/mimic-relative-positioning-in-svg/

**Comment by marasva** [comment]
> 
> 
> Seems overly complex to import and place every icon freely one by one. I am no expert on svgs but could we solve this by making a nested svg instead of handling each of them separately?
> 
> This article seems interesting: https://www.sarasoueidan.com/blog/mimic-relative-positioning-in-svg/

Will look into the article you sent.

**Review by JonatanPaalsson — APPROVED** [review]
LGTM!

**Pull request merged**